File: ChatApp/Login/Steps - JavaScript/StepDefinition-Login-Facebook_Account.js
module.exports = function () {

  this.Given(/^a user with the username: \"([^\"]*)\"$/, function (firstnamelastname, callback) {
    callback.pending();
  });

  this.Given(/^I use the username: \"([^\"]*)\"$/, function (firstnamelastname, callback) {
    callback.pending();
  });

  this.Given(/^I use the username: (.+)$/, function (username, callback) {
    callback.pending();
  });

  this.When(/^I attempt to login$/, function (callback) {
    callback.pending();
  });

  this.Then(/^I should be given access to the service$/, function (callback) {
    callback.pending();
  });

  this.Then(/^I should be informed that the password I have entered is incorrect$/, function (callback) {
    callback.pending();
  });

  this.Then(/^I should be informed that: \"([^\"]*)\"$/, function (theemailaddressorphonenumberthatyouveentereddoesntmatchanyaccountsignupforanaccount, callback) {
    callback.pending();
  });

  this.And(/^a password: \"([^\"]*)\"$/, function (shortpass, callback) {
    callback.pending();
  });

  this.And(/^I use the password: \"([^\"]*)\"$/, function (shortpass, callback) {
    callback.pending();
  });

  this.And(/^I use the password: (.+)$/, function (password, callback) {
    callback.pending();
  });

};

File: ChatApp/Login/Steps - Ruby/StepDefinition-Login-Facebook_Account.rb
Given /^a user with the username: \"([^\"]*)\"$/" do |firstnamelastname|
    # do something
end

Given /^I use the username: \"([^\"]*)\"$/" do |firstnamelastname|
    # do something
end

Given /^I use the username: (.+)$/ do |username|
    # do something
end

When /^I attempt to login$/ do 
    # do something
end

Then /^I should be given access to the service$/ do 
    # do something
end

Then /^I should be informed that the password I have entered is incorrect$/ do 
    # do something
end

Then /^I should be informed that: \"([^\"]*)\"$/" do |theemailaddressorphonenumberthatyouveentereddoesntmatchanyaccountsignupforanaccount|
    # do something
end

And /^a password: \"([^\"]*)\"$/" do |shortpass|
    # do something
end

And /^I use the password: \"([^\"]*)\"$/" do |shortpass|
    # do something
end

And /^I use the password: (.+)$/ do |password|
    # do something
end

